Package.swift

```diff
@@ -13,24 +13,32 @@ let package = Package(
   ],
   products: [
-    .library(name: "SwiftBasicFormat", targets: ["SwiftBasicFormat"]),
-    .library(name: "SwiftCompilerPlugin", targets: ["SwiftCompilerPlugin"]),
-    .library(name: "SwiftDiagnostics", targets: ["SwiftDiagnostics"]),
-    .library(name: "SwiftIDEUtils", targets: ["SwiftIDEUtils"]),
-    .library(name: "SwiftIfConfig", targets: ["SwiftIfConfig"]),
-    .library(name: "SwiftLexicalLookup", targets: ["SwiftLexicalLookup"]),
-    .library(name: "SwiftOperators", targets: ["SwiftOperators"]),
-    .library(name: "SwiftParser", targets: ["SwiftParser"]),
-    .library(name: "SwiftParserDiagnostics", targets: ["SwiftParserDiagnostics"]),
-    .library(name: "SwiftRefactor", targets: ["SwiftRefactor"]),
-    .library(name: "SwiftSyntax", targets: ["SwiftSyntax"]),
-    .library(name: "SwiftSyntaxBuilder", targets: ["SwiftSyntaxBuilder"]),
-    .library(name: "SwiftSyntaxMacros", targets: ["SwiftSyntaxMacros"]),
-    .library(name: "SwiftSyntaxMacroExpansion", targets: ["SwiftSyntaxMacroExpansion"]),
-    .library(name: "SwiftSyntaxMacrosTestSupport", targets: ["SwiftSyntaxMacrosTestSupport"]),
-    .library(name: "SwiftSyntaxMacrosGenericTestSupport", targets: ["SwiftSyntaxMacrosGenericTestSupport"]),
-    .library(name: "_SwiftCompilerPluginMessageHandling", targets: ["SwiftCompilerPluginMessageHandling"]),
-    .library(name: "_SwiftLibraryPluginProvider", targets: ["SwiftLibraryPluginProvider"]),
+    .library(name: "SwiftBasicFormat", type: libraryType, targets: ["SwiftBasicFormat"]),
+    .library(name: "SwiftCompilerPlugin", type: libraryType, targets: ["SwiftCompilerPlugin"]),
+    .library(name: "SwiftDiagnostics", type: libraryType, targets: ["SwiftDiagnostics"]),
+    .library(name: "SwiftIDEUtils", type: libraryType, targets: ["SwiftIDEUtils"]),
+    .library(name: "SwiftIfConfig", type: libraryType, targets: ["SwiftIfConfig"]),
+    .library(name: "SwiftLexicalLookup", type: libraryType, targets: ["SwiftLexicalLookup"]),
+    .library(name: "SwiftOperators", type: libraryType, targets: ["SwiftOperators"]),
+    .library(name: "SwiftParser", type: libraryType, targets: ["SwiftParser"]),
+    .library(name: "SwiftParserDiagnostics", type: libraryType, targets: ["SwiftParserDiagnostics"]),
+    .library(name: "SwiftRefactor", type: libraryType, targets: ["SwiftRefactor"]),
+    .library(name: "SwiftSyntax", type: libraryType, targets: ["SwiftSyntax"]),
+    .library(name: "SwiftSyntaxBuilder", type: libraryType, targets: ["SwiftSyntaxBuilder"]),
+    .library(name: "SwiftSyntaxMacros", type: libraryType, targets: ["SwiftSyntaxMacros"]),
+    .library(name: "SwiftSyntaxMacroExpansion", type: libraryType, targets: ["SwiftSyntaxMacroExpansion"]),
+    .library(name: "SwiftSyntaxMacrosTestSupport", type: libraryType, targets: ["SwiftSyntaxMacrosTestSupport"]),
+    .library(
+      name: "SwiftSyntaxMacrosGenericTestSupport",
+      type: libraryType,
+      targets: ["SwiftSyntaxMacrosGenericTestSupport"]
+    ),
```

Comment:
IIRC this should be formatted as `.library(name: "SwiftSyntaxMacrosGenericTestSupport"` so `swift-syntax-dev-utils` work correctly

Reply:
How annoying. I renamed `libraryType` to `type` to make it fit on a line 😄